src/test/resources/TestCases.json

```diff
@@ -0,0 +1,86 @@
+[
+  {
+    "fileName": "/home/tony_and_dj/en_ulb_gen_chunk.tr",
+    "languageCode": "en",
+    "dublinCoreId": "ulb",
+    "projectId": "gen",
+    "mediaExtension": "mp3",
+    "mediaQuality": "hi",
+    "grouping": "chunk",
+    "expectedResult": "en/ulb/gen/CONTENTS/tr/mp3/hi/chunk/en_ulb_gen_chunk.tr"
```

Comment:
I recommend adding a field here perhaps called `expectedException`.  If there is no exception expected it can be left blank or omitted.  The test harness can then use this field to determine whether to expect an exception or a filepath.